problem2.R

```diff
@@ -0,0 +1,58 @@
+# Problem 2 - Git and Github Assignment
+
+# Load required packages ---
+library(tidyverse)
+# ---
+
+# Load raw data ---
+raw_data <- readLines("suites_dw_Table1.txt")
+# ---
+
+# Saving the variable descriptions ---
+
+# Locate the separation row between variable descriptions and the values
+L <- which(raw_data == " ")
+
+# Save variable descriptions in a variable
+var_descr <- raw_data[1:(L-1)]
+
+# Save the variable descriptions in a text file
+cat(
+  var_descr,
+  file = "variable_description.txt",
+  sep = ";"
+)
+# ---
+
+# Read data to a tidy data frame ---
+
+# Save the part of raw data containing values of interest
```

Comment:
Here, you have saved all the data, including the data values, the column names and the separator line looking like this "-----+----". 

To avoid saving the separator line, you could have saved one vector with the column names (L+1), and one vector with data values (L+2):length(data). Next step would be to add these vector together, and then the seperator line would have been excluded.